index.js

```diff
@@ -26,6 +26,7 @@ if (!process.env.token) {
 let appName = 'MyApp';
```

Comment:
Can we move all these to the config file instead so commands read them from config instead of having to pass them in through the module constructor? Re appName, iconUrl, channel IDs. Also I believe passing the channel name is sufficient #random, #general, etc., no need to use the channel ID.